packages/devtools-proxy-support/src/ssh.ts

```diff
@@ -92,7 +92,7 @@ export class SSHAgent extends AgentBase implements AgentWithInitialize {
     this.logger.emit('ssh:establishing-conection', {
       host: sshConnectConfig.host,
       port: sshConnectConfig.port,
-      password: !!sshConnectConfig.passphrase,
+      password: !!sshConnectConfig.password,
```

Comment:
😭

Reply:
To be fair, this bit is kinda unrelated, but still, yeah, not my smartest move 😅

Reply:
Yeah I see now it is just in a logging event 👍